css/style.css

```diff
@@ -0,0 +1,648 @@
+  align-items: center;
+  color: #000;
+}
+
+.skills {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.skills p {
+  left: 12px;
+  right: 78px;
+  top: calc(50% - 24px / 2);
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  margin-right: auto;
+  align-items: center;
+  color: #000;
+}
+
+.arrow-img {
+  transform: rotate(-90deg);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-right: 40px;
+  padding-top: 30px;
+  width: 12px;
+  height: 6px;
+}
+
+.form-container {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+}
+
+.form-min {
+  background-color: #6070ff;
+  border-top-left-radius: 70px;
+  background-image: url("../img/form.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  padding: 50px 24px 0 24px;
+  padding-top: 103px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.form-des p {
+  width: 327px;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+  align-items: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.form-des h1 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #fff;
+  margin-top: 103px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+#user-name,
+#email-address {
+  width: 327px;
+  height: 48px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 0 0 0 17px;
+}
+
+#user-name ::placeholder,
+#email-address ::placeholder {
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  margin-left: 3px;
+}
+
+#msg {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 170px;
+  padding: 18px 0 0 17px;
+}
+
+.shape {
+  margin-left: 120px;
+  margin-bottom: 8px;
+}
+
+hr {
+  flex: 0 0 100%;
+}
+
+@media screen and (max-width: 768px) {
+  .nav-ul a {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 900px) {
+  .work-img1 img,
+  .work-img2 img,
+  .work-img3 img,
+  .work-img4 img {
+    width: 85%;
+    height: 70%;
+  }
+}
+
+@media screen and (max-width: 980px) {
+  .work-img1 img,
+  .work-img2 img,
+  .work-img3 img,
+  .work-img4 img {
+    width: 87%;
+    height: 75%;
+  }
+}
+
+@media screen and (min-width: 980px) {
+  #home {
+    padding: 160px 260px 90px 130px;
+    height: 70vh;
+    text-align: left;
+    border-bottom-left-radius: 20%;
+    background-color: #fff;
+  }
+}
+
+@media screen and (max-width: 1200px) {
+  .work-img1 img,
+  .work-img2 img,
+  .work-img3 img,
+  .work-img4 img {
+    width: 95%;
+    height: 95%;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  #home {
+    background-image: url(../img/home_bg.svg);
+    background-repeat: no-repeat;
+    background-size: 100%;
+  }
+
+  .home p {
+    width: 100%;
+  }
+
```

Comment:
[OPTIONAL] 👍🏽 
- I think it will be better if you increase the font size of your `.home h1` on the desktop screen as it currently still maintains the 40px you used on your mobile version. 🍷